api-specs/friend.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package friendsapi;
+option go_package = "/friendsapi"
+
+import "google/protobuf/empty.proto";
+
+service FriendService {
+	rpc Add (FriendRequest) returns (google.protobuf.Empty);
+	rpc Accept (FriendRequest) returns (google.protobuf.Empty);
+	rpc Block (FriendRequest) return (google.protobuf.Empty);
```

Comment:
sorry what does Block FriendRequest do

Reply:
It's our block a user, but not sure if we gonna have time to implement right :')